matchms/Pipeline.py

```diff
@@ -95,18 +93,40 @@ def _initialize_workflow_dict(self, config_file):
             self.workflow = OrderedDict()
             self.workflow["importing"] = {"queries": None,
                                           "references": None}
-            self.workflow["filtering_queries"] = ["default_filters"]
-            self.workflow["filtering_refs"] = ["default_filters"]
+            self.workflow["predefined_processing_queries"] = "default"
```

Comment:
I did not directly get that refs stood for reference. So maybe change that to additional_processing_references

Reply:
Thanks. I will change this.